integration-tests/support/commands/hooks.ts

```diff
@@ -17,7 +17,7 @@ before(() => {
     JSON.stringify({ 'application-list-getting-started-modal': true }),
   );
 
-  if (Cypress.env('PR_CHECK') === true) {
+  if (Cypress.env('PR_CHECK') || Cypress.env('PERIODIC_RUN')) {
```

Comment:
`Cypress.env('PR_CHECK')` is going to have a default value of `'false'`, which in this case is still going to produce a truthy result.
In this configuration it pretty much always acts as if it's a pr check.

Reply:
when `Cypress.env('PERIODIC_RUN')` is `true` I want to execute this if block for ephemeral env to login

Reply:
Update the `PR_CHECK` to boolean in cofig